### DISCUSSION TITLE
Версия 2.2.5

### DISCUSSION THREAD

**Discussion by lirix360** [lirix360/ReadmangaGrabber#108] [Announcements]
Изменения:

- Исправления и оптимизации.

Сообщать об ошибках и предложениях [можно в обсуждениях](https://github.com/lirix360/ReadmangaGrabber/discussions).

<hr /><em>This discussion was created from the release <a href='https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.2.5'>Версия 2.2.5</a>.</em>

**Comment by mosbug** [comment]
с readmanga вообще ничего не скачивает, выходят ошибки постоянно

**Reply by lirix360** [reply]
Можно примеры того что не качается и ошибок?

**Reply by VectorElex** [reply]
С mangalib.me тоже перестало качать.

**Comment by Kristinaanime** [comment]
С readmanga скачивается, а вот с mintmanga нет

Вот это https://2.mintmanga.one/istoriia_isimi

![image](https://github.com/user-attachments/assets/c204e3ba-014b-4a36-8af0-d4761095ee0a)

**Comment by TheShowBeginsNow** [comment]
![Новый точечный рисунок](https://github.com/user-attachments/assets/6a52cb3d-f5e0-4d01-b12f-917359698b49)
![Новый точечный рисунок1](https://github.com/user-attachments/assets/ecec0638-838f-4b5b-b05e-cfa93784cfb1)

Выдает ошибку при получении списка глав. Для любой манги, не только для той что на скрине. Сперва перестала работать старая версия, скачал новую, проблема та же.

**Reply by TheShowBeginsNow** [reply]
А вот сейчас проверил, с mintmanga все работает, находит главы и скачивает без проблем, а с mangalib такая же беда, не находит главы. Даже те манги которые уже когда то скачивал, ту же ошибку выдает.

**Comment by GithubPapito** [comment]
Доброго времени суток, не качает мангу с 1.seimanga.me, окно консоли просто крашит, в логах ошибок нет. Для примера: пробую скачать эту мангу https://1.seimanga.me/grabitel.
С web.usagi.one всё качает.

**Reply by MorasuI** [reply]
Согласен. Такая же проблема. Только ещё недели 2-3 назад скачивало с сайта 1.seimanga.me, а эти дни вдруг перестало. Теперь вылетает при попытке скачивания...

**Comment by FarongKh** [comment]
Добрый день, при попытке скачать с web.usagi.one вылетает при нажатии кнопки скачать. Пытался и в корень диска кидать, и имя оригинальное оставлять, и с исходников компилировать, и на разных компьютерах (Win 10/11). В логах только это, будто кнопка "Скачать выбранные главы" это кнопка закрытия:

>time=2025-01-04T04:44:29.216+03:00 level=INFO msg="Запуск приложения"

**Comment by mark3701** [comment]
скачал последний релиз, пытаюсь скачать с https://1.seimanga.me/ или же с https://web.usagi.one/ - как только произвожу поиск и нажимаю "скачать все главы" затем командная строка закрывается и просто висит скачивание 1 тома и 1 главы без загрузки вообще :( в логах только открытие приложения

**Comment by YokitaHoshimori** [comment]
Друг, как видишь, перестало работать:(
Твоя утилита мне очень нужна, живых аналогов для скачивания с русских сайтов просто не существует, я бы купил даже, если бы продавалась

**Reply by Aforege** [reply]
Поддерживаю! Надеюсь, что автор услышит нас

**Comment by Aforege** [comment]
Уважаемый автор! Пожалуйста, обновите программу, очень хочется продолжить скачивать мангу с сайта web.usagi.one. Можно встроить какое-нибудь окошко для донатов или сделать программу платной. Кроме вас никто не сможет!

**Reply by TheShowBeginsNow** [reply]
Поддерживаю!
